qdrant-landing/content/documentation/concepts/indexing.md

```diff
@@ -315,27 +315,18 @@ performance.
 
 *Available as of v1.7.0*
 
-### Key Features of Sparse Vector Index
-- **Support for Sparse Vectors:** Qdrant supports sparse vectors, characterized by a high proportion of zeroes.
-- **Efficient Indexing:** Utilizes an inverted index structure to store vectors for each non-zero dimension, optimizing memory and search speed.
+Qdrant supports a separate index for Sparse Vectors. Each data point in Qdrant can have both dense and sparse vectors. Therefore, you can still use the same collection for both vector types.  The indexing process is highly efficient, whereby an inverted index structure stores vectors for each non-zero dimension. This approach reduces memory usage and increases search speeds, particularly for high-dimensional data. 
 
-### Search Mechanism
-- **Index Usage:** The index identifies vectors with non-zero values in query dimensions during a search.
-- **Scoring Method:** Vectors are scored using the dot product.
+During search, the index identifies vectors with non-zero values in query dimensions. They are scored using the dot product method. You may also use filtering to search through sparse vectors. Just as with dense vectors, filtering is done by payload.
```

Comment:
> During search, the index identifies vectors with non-zero values in query dimensions.

I am not sure this is a best way to describe what happens.

Reply:
We can be more specific. (Is it "the same" or just "similar" query dimensions?)

```suggestion
During search, the index identifies vectors with non-zero values in the same query dimensions. They are scored using the dot product method. You may also use filtering to search through sparse vectors. Just as with dense vectors, filtering is done by payload.
```